routes/index.js

```diff
@@ -64,7 +64,7 @@ module.exports = (keystone) => {
 				return next();
 			}
 
-			res.render('job', job);
+			res.redirect(job.url_base);
```

Comment:
Continuing to play with this. I'm finding that dealing with job listings without urls is making this all more complicated.

Since jobs without a job listing url from the employer require that I still make the job page template for our site, I'm now experimenting with simplifying our job listing page and making it more flexible by just having a big text box the employer can use in any way they see fit.

I'll just ask the employer for a summary to be shown on the `/jobs` page and then optionally they can add details to that summary on the `/jobs/:slug` page.